pkg/resources/map_utils.go

```diff
@@ -0,0 +1,22 @@
+package resources
+
+func MergeMapStringString(existing *map[string]string, desired map[string]string) bool {
```

Comment:
nit: might be nice to add a quick comment so its known the bool returned is to let you know if the map was modified?

Reply:
done